### PR TITLE
add phone icon & disable news label if no news

### PR DESCRIPTION
**Pull request by furkan7258**
- Added the telephone icon.
- Added condition not to show news label on home page if there are no news.

### CODE REVIEW

**Review by doganulus — CHANGES_REQUESTED** [review]
Two comments:
1. It is better if the `if` block contains the whole `<section>`/`</section>` block.
2. Use a more descriptive variable name such as `enable_news`. Make it configurable from the Hugo site `config/_default/params.yml`. Make the default value `true`.

**Comment by furkan7258** [comment]
> 1. It is better if the `if` block contains the whole `<section>`/`</section>` block.

I think this causes the layout to change when there are news or not. Events would be shown on the left when there are news, on the right otherwise.

---

> 2. Use a more descriptive variable name such as `enable_news`. Make it configurable from the Hugo site `config/_default/params.yml`. Make the default value `true`.

I will add this, as per the issue #2, as extra to the currently proposed condition of `$news`. The committed condition here is just like the one for the events.

File: layouts/partials/newsroom.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{{ $news := where site.RegularPages.ByDate.Reverse "Section" "news" }}
+<section id="newsroom" class="col-xl-8">
```
Comment (doganulus) [inline layouts/partials/newsroom.html]: Remove `class="col-xl-8"` here and call the partial inside a new div block with `class="col-xl-8"` on the index page.

File: layouts/index.html [hunk 2]
```diff
@@ -15,44 +14,19 @@
...
-                {{ range $index, $element := first 6 (after 5 $news) }}
-                <div class=" col">
-                    {{ partial "news/news-card.html" . }}
-                </div>
-                {{ end }}
-            </section>
-        </section>
+        {{ if site.Params.newsroom.enabled }}
+            {{ partial "newsroom.html" . }}
+        {{ end }}
```
Comment (doganulus) [inline layouts/index.html]: Wrap it with a `div` block having `class="col-xl-8"`.

File: layouts/index.html [hunk 3]
```diff
@@ -15,44 +14,19 @@
     <div class="row">
         <section id="event-board" class="col-xl-4 order-xl-5 mb-5">
```
Comment (doganulus) [inline layouts/index.html]: Let's add an `eventboard` partial as well.

**Comment by furkan7258** [comment]
The changes don't work currently, I'll handle it soon.

**Comment by furkan7258** [comment]
It turns out they work but the config needs to be updated in the `website` side: `bouncmpe/website`.

**Comment by doganulus** [comment]
@furkanakkurt1335 See my comments in the review.

**Comment by furkan7258** [comment]
`newsroom` partial is created and configuration is done in the `hugo.yml` file which seems to be the default name in the documentation. If condition now encapsulates the entire partial. Which comment do you mean?

**Comment by doganulus** [comment]
> `newsroom` partial is created and configuration is done in the `hugo.yml` file which seems to be the default name in the documentation. If condition now encapsulates the entire partial. Which comment do you mean?

Here one can leave comments on the code line by line:
https://github.com/bouncmpe/hugo-theme-department/pull/1/files/1d476239bb866909d9dc93a5234e92efec2e18eb

**Comment by furkan7258** [comment]
Yes, but I cannot see them, sorry 😅

**Comment by doganulus** [comment]
Or this one: https://github.com/bouncmpe/hugo-theme-department/pull/1#pullrequestreview-1516020588

**Comment by furkan7258** [comment]
I can't see the review on the codes. I have seen your 2 comments, replied to them and implemented them.

I don't know how I can't see them, maybe a permission issue?

**Comment by doganulus** [comment]
> I can't see the review on the codes. I have seen your 2 comments, replied to them and implemented them.
> 
> I don't know how I can't see them, maybe a permission issue?

I added inline comments like this one:

> Remove class="col-xl-8" here and call the partial inside a new div block with class="col-xl-8" on the index page.

You must be able to see them on this thread and in the review (files changed).

**Review by doganulus — CHANGES_REQUESTED** [review]

**Comment by furkan7258** [comment]
Only now am I able to see them, even though they are dated 2 days ago, it's weird. Let me have a look now.

File: layouts/partials/newsroom.html (hunk 1 above)
Comment (furkan7258) [reply layouts/partials/newsroom.html]: done!

File: layouts/index.html (hunk 2 above)
Comment (furkan7258) [reply layouts/index.html]: done!

File: layouts/index.html (hunk 3 above)
Comment (furkan7258) [reply layouts/index.html]: done!

**Pull request merged**